docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,76 @@ To set these configuration options, create a ``Config`` object with the options
 
     client = boto3.client('kinesis', config=my_config)
 
+
+Using proxies
+~~~~~~~~~~~~~
+Boto3 supports using proxies as intermediaries between your code and AWS. Proxies may provide any number of functions, from filtering to security and firewalls to privacy assurance.
+
+
+.. _specify_proxies:
+
+Specifying proxy servers
+''''''''''''''''''''''''
+
+You can specify proxy servers to be used for connections when using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary in which each entry maps a protocol to the address and port number of the proxy server for that protocol.
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_definitions = {
+        'http': 'http://proxy.amazon.com:6502',
+        'https': 'https://proxy.amazon.org:2010'
+    }
+
+    my_config = Config(
+        'region_name': 'us-east-2',
+        'signature_version': 'v4',
+        'proxies': proxy_definitions
+    }
+
+    client = boto3.client('kinesis', config=my_config)
+
+
+In the above example, a proxy list is set up to use ``proxy.amazon.com``, port 6502 as the proxy for all HTTP requests by default. HTTPS requests use port 2010 on ``proxy.amazon.org`` instead.
+
+
+.. _configure_proxies:
+
+Configuring proxies
+'''''''''''''''''''
+You can configure proxy usage with the ``proxies_config`` option, which is a dictionary that specifies the values of several proxy options by name.  There are three keys in this dictionary: ``proxy_ca_bundle``, ``proxy_client_cert``, and ``proxy_use_forwarding_for_https``. See the `Botocore config reference <https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html#botocore.config.Config>`_ for more information about these keys.
```

Comment:
You can configure how Boto3 uses proxies by using the ''proxies_config'' option...

Last sentence: For more information about these keys, see ...